pages/governance/proposing_a_new_market.md

```diff
@@ -0,0 +1,130 @@
+| 3 | [MsgDelayMessage](https://github.com/dydxprotocol/v4-chain/blob/35b87db422b0ef4138101ba73b0f00d16780ba89/proto/dydxprotocol/delaymsg/tx.proto#L18) | Transitions the orderbook created by `MsgCreateClobPair` to `ACTIVE` status, after some amount of blocks| [delay_blocks](https://github.com/dydxprotocol/v4-chain/blob/35b87db422b0ef4138101ba73b0f00d16780ba89/proto/dydxprotocol/delaymsg/tx.proto#L27) |
+
+Notes:
+
+- For guideline on choosing values for individual params, see next [section](./proposing_a_new_market.md#choosing-market-parameters).
+- The exact ordering of messages above is necessary for successful onchain execution.
+- Each of the 4 top-level messages should have `Authority = dydx10d07y265gmmuvt4z0w9aw880jnsr700jnmapky`, the [gov module](https://github.com/dydxprotocol/v4-chain/blob/5e72896719e2f8d2fe6e10fddbde18b363a6bbe3/protocol/app/module_accounts_test.go#L28).
+- The `MsgCreateClobPair` message wrapped in `messages[3]: MsgDelayMessage` should have `Authority = dydx1mkkvp26dngu6n8rmalaxyp3gwkjuzztq5zx6tr`, the [delaymsg module](https://github.com/dydxprotocol/v4-chain/blob/5e72896719e2f8d2fe6e10fddbde18b363a6bbe3/protocol/app/module_accounts_test.go#L36).
+- The identifier fields must be consistent for a perpetual market: `params.id`, `params.market_id`, `clob_pair.id`, `clob_pair.perpetual_clob_metadata.perpertual_id`.  An `id` value is valid as long as it's a `uint32` unique from existing markets (they do not need to follow existing market ids).
+- `MsgDelayMessage.delay_blocks` is recommended to be `3600` blocks (`1 hour` at `1 sec` blocktime).
+
+## Choosing Values for Market Parameters
+
+The following decribes how to set various parameters for a new market and assumes that the market listed is:
+
+- Safe (Not susceptible to market manipulation e.g. [Mango](https://www.notion.so/601722959e6d42ad8636f0e827885e9f?pvs=21))
+- Worth listing (Will this generate sufficient trading volume to consume throughput?)
+- Oracle sources are known (Which spot exchanges should be selected when determining the oracle price?)
+
+### Inputs
+
+- Ticker Symbol (e.g. `BTC-USD` )
+- Reference price (e.g. `40,000`) and `p` value
+  - Defines the exponent on the reference price as `log_price:=FLOOR(log10(reference_price))`. For example `log_price:=FLOOR(log10(40,000))=4`
+- Exchanges required for oracles (`exchange_config_json`)
+- Liquidity Tier (these are based on the liquidity tiers recommended, so mappings could change if these tiers change)
+  - 0: Large Cap (BTC/ETH)
+  - 1: Mid-Cap
+  - 2: Long-tail
+
+### Outputs
+
+| Message Type | Field | Description | Value |
+|--------------|-------|-------------|-------|
+| `MsgCreateOracleMarket` | `exponent` | TODO: definition | `p-9` |
```

Comment:
Definition was missing from research doc

Reply:
Definition: `Exponent denotes the number of decimals a value should be shifted in the price daemon.`